airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -243,11 +244,15 @@ def run(
 
     @staticmethod
     def _make_serializable(result):
-        """Transform the databricks Row objects into JSON-serializable lists."""
+        """Transform the databricks Row objects into JSON-serializable namedtuple."""
```

Comment:
Please remove the JSON remark. It is either XCom serializable, serde serializable or just serializable. JSON has no meaning here.

Reply:
> I wonder btw if we should not have a typedef for it if this is part the the DbAPI. That would make things conceptually stronger and less ambiguous.

Yes. It was missing from the beginning. Agree JSON in the description is wrong.

Reply:
Thanks for the re-scoping ! I tried to be reactive on this one, but did not put enough care in the namings. I updated the code and the PR message.